packages/cdrkit.rb

```diff
@@ -25,7 +25,7 @@ class Cdrkit < Package
 
   def self.build
     system 'make clean'
-    system 'make CFLAGS=-I/usr/local/include LDFLAGS=-L/usr/local/lib'
+    system 'make CFLAGS=-I#{CREW_PREFIX}include LDFLAGS=-L#{CREW_LIB_PREFIX}'
```

Comment:
`-I#{CREW_PREFIX}/include`.  The constant doesn't include the trailing slash.  Otherwise, the correct fix.